### PR TITLE
Fix many spell glitches in docstrings and comments

### PR DESCRIPTION
**Pull request by lelit**
These were revealed by running `pylint --disable all --enable spelling --spelling-dict en_US` over all sources.

### CODE REVIEW

**Review by CaselIT — APPROVED** [review]
Thanks.

I think we could add a job in the github actions

**Review by sqla-tester — COMMENTED** [review]
OK, this is **sqla-tester** setting up my work on behalf of **CaselIT** to try to get revision 0ee08659935326573805da391c074852e8ad1bf3 of this pull request into gerrit so we can run tests and reviews and stuff

**Comment by sqla-tester** [comment]
New Gerrit review created for change 0ee08659935326573805da391c074852e8ad1bf3: https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/2507

**Comment by lelit** [comment]
I have an uncommitted "checked-words", and also a set of "ignore-words", but still lot of noise comes out, so I'm not sure it's effective to have an automated check...
I could not understand the _failure_ on Gerrit, please let me know if there's something I can do.

**Comment by CaselIT** [comment]
for pep8 you have to look at the console output. It looks like a couple of lines are too long
```
./lib/sqlalchemy/orm/query.py:774:80: E501 line too long (80 > 79 characters)
./lib/sqlalchemy/orm/query.py:953:80: E501 line too long (80 > 79 characters)
```

**Comment by CaselIT** [comment]
> I have an uncommitted "checked-words", and also a set of "ignore-words", but still lot of noise comes out, so I'm not sure it's effective to have an automated check...

Yes, I guess a lot of words would need allewed/ignored, like all the dbapis etc.

**Comment by lelit** [comment]
> I have an uncommitted "checked-words", and also a set of "ignore-words", but still lot of noise comes out, so I'm not sure it's effective to have an automated check...

For the record, here they are: https://gist.github.com/lelit/091542cfcd1ed06950181a4f7ec1ea90

**Review by sqla-tester — COMMENTED** [review]
OK, this is **sqla-tester** setting up my work on behalf of **CaselIT** to try to get revision bb249195d92e3b806e81ecf1192d5a1b3cd5db48 of this pull request into gerrit so we can run tests and reviews and stuff

**Comment by sqla-tester** [comment]
Patchset bb249195d92e3b806e81ecf1192d5a1b3cd5db48 added to existing Gerrit review https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/2507

**Comment by sqla-tester** [comment]
**Federico Caselli** (CaselIT) wrote:

Code-Review+1 Workflow+2

I think this can be merged

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/2507

File: lib/sqlalchemy/engine/result.py [hunk 1]
```diff
@@ -1709,7 +1709,7 @@ def __init__(self, cursor_metadata, results):
         self._unique_filter_state = results[0]._unique_filter_state
         self._yield_per = results[0]._yield_per
 
-        # going to try someting w/ this in next rev
+        # going to try something w/ this in next rev
```
Comment (sqla-tester) [inline lib/sqlalchemy/engine/result.py:1712]: **mike bayer** (zzzeek) wrote:

I wonder what I was going to try? :)

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/2507

**Review by sqla-tester — COMMENTED** [review]
**mike bayer** (zzzeek) wrote:

Code-Review+2 Workflow+2

(1 comment)

excellent, thanks so much for this!!!    +1000

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/2507

**Comment by sqla-tester** [comment]
Gerrit review https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/2507 has been **merged**. Congratulations! :)

**Comment by lelit** [comment]
My pleasure!

**Pull request closed without merging**